src/game/client/shadow/w3dvolumetricshadow.h

```diff
@@ -0,0 +1,157 @@
+/**
+ * @file
+ *
+ * @author tomsons26
+ *
+ * @brief
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#pragma once
+
+#include "always.h"
+#include "hash.h"
+#include "refcount.h"
+#include "vector3.h"
+#include "vector3i.h"
+#include <new>
+
+class RenderObjClass;
+class W3DShadowGeometry;
+class MeshClass;
+class Vector3;
+
+typedef Vector3i16 TriIndex;
+
+enum
+{
+    MAX_SHADOW_CASTER_MESHES = 160,
+};
+
+#pragma pack(push, 2)
+
+struct NeighborEdge
+{
+    short neighborIndex;
```

Comment:
Are you sure this is correct. Everything seems to suggest that there is 16bits before this field.